tests/dummy/app/router.js

```diff
@@ -23,13 +23,14 @@ Router.map(function() {
   });
 
   this.route('cookbook', function() {
+    this.route('/');
```

Comment:
I think you mean `this.route('index');` here? If so, you can skip that, since the `index` route is created implicitly.

Reply:
Correct, removed as requested